nixos/lib/test-driver/nixos_test_driver/nixos_test_driver.py

```diff
@@ -131,6 +139,8 @@ def retry(fn: Callable) -> None:
 
 
 class Logger:
+    """Logger class."""
```

Comment:
I understand that these fields are needed and it is an extra-burden to document undocumented things just to make sphinx work. The text here is however completely redundant and it would be awesome to have an actual explanation how/what/why this exists/works. If you feel this is too much for this pull request, we should make it a ticket.

Reply:
The text is indeed useless and is merely for getting something in shown in Sphinx. Writing docstrings is out of scope.